src/Schema/SQLiteSchemaManager.php

```diff
@@ -151,6 +151,16 @@ static function (array $a, array $b): int {
             $idx['primary']    = false;
             $idx['non_unique'] = ! $tableIndex['unique'];
 
+            if ($tableIndex['partial'] === 1) {
+                $idx['where'] = $this->connection->fetchOne(
+                    <<<'SQL'
+                    SELECT SUBSTR(sql, INSTR(sql, 'WHERE') + 6)
```

Comment:
```suggestion
                    SELECT SUBSTR(sql, INSTR(sql, 'WHERE') + LENGTH('WHERE '))
```

Question: Is this query somehow affected by the collation of a database so that an all uppercase WHERE would end up in a wrong result?